cli/azd/pkg/keyvault/keyvault.go

```diff
@@ -356,27 +356,45 @@ func NewAkvs(subId, vaultId, secretName string) string {
 }
 
 func (kvs *keyVaultService) SecretFromAkvs(ctx context.Context, akvs string) (string, error) {
+	parseAkvs, err := ParseAkvs(akvs)
+	if err != nil {
+		return "", err
+	}
+	subscriptionId, vaultName, secretName := parseAkvs.SubscriptionId, parseAkvs.VaultName, parseAkvs.SecretName
+	// subscriptionId is required by the Key Vault service to figure the TenantId for the
+	// tokenCredential. The assumption here is that the user has access to the Tenant
+	// used to deploy the app and to whatever Tenant the Key Vault is in. And the tokenCredential
+	// can use any of the Tenant ids.
+	secretValue, err := kvs.GetKeyVaultSecret(ctx, subscriptionId, vaultName, secretName)
+	if err != nil {
+		return "", fmt.Errorf("fetching secret value from key vault: %w", err)
+	}
+	return secretValue.Value, nil
+}
+
+type Akvs struct {
+	SubscriptionId string
+	VaultName      string
+	SecretName     string
+}
+
+func ParseAkvs(akvs string) (Akvs, error) {
```

Comment:
I'd recommend spelling out `Akvs` since users may not be immediately aware that it refers to an Azure KeyVault Secret reference.